packages/web-worker/src/notifier.js

```diff
@@ -31,7 +31,9 @@ export const Bugsnag = {
 
     bugsnag._logger.debug('Loaded!')
 
-    return bugsnag
+    return bugsnag._config.autoTrackSessions
+      ? bugsnag.startSession()
+      : bugsnag
```

Comment:
I don't think this works since the `@bugsnag/plugin-browser-session` isn't installed for `@bugsnag/web-worker` — there's no `_sessionDelegate` on `bugsnag` here, so it'll crash in core

This could use some unit & end to end tests 😀